content/applications/hr/appraisals.rst

```diff
@@ -589,3 +519,11 @@ Each skill has the following information listed:
 .. image:: appraisals/skills-report.png
    :align: center
    :alt: A report showing all the skills grouped by employee.
+
+.. seealso::
+   - :doc:`appraisals/goals`
```

Comment:
I don't believe we need a bullet point when there's just one link in the see also (:
```suggestion
   :doc:`appraisals/goals`
```

Reply:
Force of habit! I always use this when there are multiple docs =D - which there will be soon!